bedrock/mozorg/templates/mozorg/home/home-new.html

```diff
@@ -43,8 +46,10 @@
 {% endblock %}
 
 {% block page_banner %}
-  {%- if (LANG == 'de' or LANG.startswith('en-')) and show_pocket_banner -%}
-    {% include 'includes/banners/pocket-pencil-banner.html' %}
+  {# {%- if (LANG == 'de' or LANG.startswith('en-')) and show_pocket_banner -%}
```

Comment:
Guessing this comment is an artifact from local testing? I think this needs to be uncommented and the following rule changed to `elif`.